src/java.base/share/classes/sun/util/calendar/CalendarUtils.java

```diff
@@ -106,28 +108,28 @@ public static final int floorDivide(int n, int d, int[] r) {
         return q;
     }
 
-    public static final long mod(long x, long y) {
+    public static long mod(long x, long y) {
         return (x - y * floorDivide(x, y));
     }
 
-    public static final int mod(int x, int y) {
+    public static int mod(int x, int y) {
         return (x - y * floorDivide(x, y));
     }
 
-    public static final int amod(int x, int y) {
+    public static int amod(int x, int y) {
         int z = mod(x, y);
         return (z == 0) ? y : z;
     }
 
-    public static final long amod(long x, long y) {
+    public static long amod(long x, long y) {
         long z = mod(x, y);
         return (z == 0) ? y : z;
     }
 
     /**
      * Mimics sprintf(buf, "%0*d", decaimal, width).
      */
-    public static final StringBuilder sprintf0d(StringBuilder sb, int value, int width) {
+    public static StringBuilder sprintf0d(StringBuilder sb, int value, int width) {
```

Comment:
Can this (and the following overload) method be eliminated? No need to mimic `sprintf` here, but using `String.format`/`formatted` should suffice?

Reply:
Thanks for reviewing, replaced the two methods with Formatter (there are occurrences in _test_ as well)

Reply:
Oh, I was not aware that the utility methods were used in `Date.toString()`. I will have to take my suggestion back for performance reasons (assuming the existing implementation is faster). Sorry for not noticing that beforehand.